en/SetupView/FlightModes.md

```diff
@@ -90,7 +90,7 @@ The video then shows how to use *QGroundControl* to specify the mode channel and
 {% youtube %}
 http://www.youtube.com/watch?v=scqO7vbH2jo
 {% endyoutube %}
-
+** NOTE: THE ABOVE VIDEO LINK IS BROKEN **
```

Comment:
Thanks for pointing this out. For now, we will have to just delete this.

Reply:
Actually, I have just removed your note. I'm going to try get some help to get a new image.